code/game/objects/structures/statues.dm

```diff
@@ -314,7 +331,7 @@
 
 /obj/structure/snowman
 	name = "snowman"
-	desc = "Seems someone made a snowman here."
+	desc = "A traditional 'statue' made from snow."
```

Comment:
Why?

Reply:
Returned it to the original. I agree it looked weird.